model_utilities/config.ini

```diff
@@ -9,8 +9,10 @@ output_model_name = pl-csb-model-v1
 batch_size = 8
 max_length = 128
 warmup_steps = 100
-training_steps = 1000
+num_epochs = 3
```

Comment:
Are 3 epochs comparable to 1000 steps?

Reply:
Depends on the size of the dataset, with the current one it's technically 18x more, nevertheless 1000 steps is not a good training length anyway and was chosen for unknown reason god knows when.